front/app/components/IdeaCards/IdeasWithFiltersSidebar/index.tsx

```diff
@@ -264,19 +241,6 @@ const IdeasWithFiltersSidebar = ({
     setFiltersModalOpened(false);
   }, []);
 
-  const [isCTABarVisible, setIsCTABarVisible] = useState(false);
-
-  useEffect(() => {
-    function checkCTABarVisibility() {
-      if (document.getElementById('project-cta-bar')) {
-        setIsCTABarVisible(true);
```

Comment:
Yeah I didn't feel that was necessary, since we only need to know if we ever "do" find the element :thinking:

Reply:
> So shouldn't that CSS change again if we scroll back up and the CTA bar is not attached anymore?
It could, I just don't think it would make any difference since if we've scrolled far enough up that the CTABar changes, the filter sidebar won't be sticky anyways (which is the only element that has this conditional CSS) :thinking: 

We can add it though regardless (set it back to false), it's probably better practice even if it's not something the user will feel :+1: